.travis.yml

```diff
@@ -1,36 +1,57 @@
 language: python
 python:
-  - "2.7"
-  - "3.5"
-  - "3.6"
+  - 2.7
+  - 3.4
+  - 3.5
+  - 3.6
+  # also includes 3.7 (see below)
 env:
-  - NUMPY_VERSION=""  WITH_COVERAGE=1 # environment to test with the latest version of NumPy
+  - NUMPY_VERSION=""  # environment to test with the latest version of NumPy
   - NUMPY_VERSION="<1.13"
+
+matrix:
+  # support for python 3.7 is rather awkward via Travis CI
+  # https://github.com/travis-ci/travis-ci/issues/9815
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
+      env: NUMPY_VERSION=""
+    - python: 3.7
+      dist: xenial
+      sudo: true
+      env: NUMPY_VERSION="<1.13"
```

Comment:
Is there any reason why this is only tested against numpy<1.13?  That version of numpy is about 2 years out of date ...

Reply:
For now it is mostly because we have compatibility and we don't really use any new features. Also, Emperor is compatible with this and with much newer versions, hence the other builds with `NUMPY_VERSION=""`.